### PR TITLE
Migrate existing features to files

### PR DESCRIPTION
**Pull request by brian-avery**
This migrates existing features to use the file based model system. 

* Everything in features is just a migration from issues over to features.
* Added `area` from the Istio.io feature status page
* Added all of the features from the Istio.io feature status page

### CODE REVIEW

File: features.yaml [hunk 1]
```diff
@@ -4,28 +4,255 @@ features:
...
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+  - name: "Mutual TLS Migration"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Standalone Operator"
```
Comment (linsun) [inline features.yaml:187]: Is this server side operator controller or istioctl?

File: features.yaml [hunk 2]
```diff
@@ -4,28 +4,255 @@ features:
...
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Core
+  - name: "External Control Plane"
+    level:
+     - checklist: features/external_istiod.md
+       maturity: Alpha
+       nextExpectedPromotion:
+       area: Core
+  - name: "Kubernetes: Istio Control Plane Upgrade"
```
Comment (linsun) [inline features.yaml:217]: in place or revision?

**Review by linsun — COMMENTED** [review]
LG - just 2 minor clarification qs.  This is ready to go IMHO.

File: features.yaml (hunk 2 above)
Comment (brian-avery) [reply features.yaml:217]: I think that refers to in-place upgrades because I think that revision based are alpha. Confirming though.

Comment (brian-avery) [reply features.yaml:217]: @howardjohn would you be able to comment here? The Istio feature status page isn't clear to me.

File: features.yaml (hunk 1 above)
Comment (brian-avery) [reply features.yaml:187]: This is referring to the server side [istioctl operator](https://istio.io/latest/docs/setup/install/operator/)

**Pull request merged**